packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -151,7 +157,7 @@
     z-index: 2;
 
     &:hover ~ .mdc-text-field__idle-outline {
-      border: 1px solid $mdc-text-field-outlined-hover-border;
+      border: 1px solid;
```

Comment:
Is this line necessary? Seems the same as line 136 above

Reply:
Line 136 is for the input:hover and this one is for icon:hover. 

They were both set to the same color, but I didn't want to assume that would always be the case.